### DISCUSSION TITLE
How to connect Mautic and Postal?

### DISCUSSION THREAD

**Discussion by ripon202310** [postalserver/postal#3181] [Help with using Postal]
I installed Mautic and Postal separately. However, I'm unable to connect Mautic to Postal despite creating the credentials in Postal.But when I try to connect Mautic using those credentials, the fields in Mautic look different. Please check my screenshot
![t1](https://github.com/user-attachments/assets/e0f073db-91c5-4802-9f0e-cfbca99ecebc)

Can anyone help me with how to connect Mautic and Postal? Thank you!

![t2](https://github.com/user-attachments/assets/8bf08691-9741-41c8-92ce-e4a0ac89efa9)

**Comment by willpower232** [comment]
I don't know anything about Mautic but I'm presuming the dropdown for Authorization Protocol has some other options which hopefully include SMTP?
